### PR TITLE
Add tsl-robin-map/0.6.2

### PR DESCRIPTION
**Pull request by p-podsiadly**
**tsl-robin-map/0.6.2**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/tsl-robin-map/all/test_package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, CMake
```
Comment (madebr) [inline recipes/tsl-robin-map/all/test_package/conanfile.py:2]:
```suggestion
from conans import ConanFile, CMake, tools
```

File: recipes/tsl-robin-map/all/test_package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        test_package = os.path.join("bin", "test_package")
+        self.run(test_package, run_environment=True)
```
Comment (madebr) [inline recipes/tsl-robin-map/all/test_package/conanfile.py:16]:
```suggestion
        if not tools.cross_building(self.settings):
            test_package = os.path.join("bin", "test_package")
            self.run(test_package, run_environment=True)
```

Comment (p-podsiadly) [reply recipes/tsl-robin-map/all/test_package/conanfile.py]: Thanks for pointing this out, I've updated the recipe.

File: recipes/tsl-robin-map/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        tools.get(**self.conan_data["sources"][self.version])
+
+        os.rename("robin-map-{}".format(self.version), self._source_subfolder)
+
+    def build(self):
+        pass
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy("include/tsl/*.h", dst="include/tsl", src=self._source_subfolder, keep_path=False)
```
Comment (uilianries) [inline recipes/tsl-robin-map/all/conanfile.py:25]:
```suggestion
        self.copy("*.h", dst="include", src=os.path.join(self._source_subfolder, "include"))
```

File: recipes/tsl-robin-map/all/test_package/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8)
```
Comment (uilianries) [inline recipes/tsl-robin-map/all/test_package/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

File: recipes/tsl-robin-map/all/test_package/CMakeLists.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+project(test_package)
+
+set(CMAKE_CXX_STANDARD 11)
```
Comment (uilianries) [inline recipes/tsl-robin-map/all/test_package/CMakeLists.txt:4]:
```suggestion
```

File: recipes/tsl-robin-map/all/test_package/CMakeLists.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::tsl-robin-map)
```
Comment (uilianries) [inline recipes/tsl-robin-map/all/test_package/CMakeLists.txt:10]:
```suggestion
target_link_libraries(${PROJECT_NAME} CONAN_PKG::tsl-robin-map)
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

**Pull request merged**